src/components/minor-button/minor-button.spec.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import TestRenderer from "react-test-renderer";
+import { shallow, mount } from "enzyme";
+import {
+  ButtonProps,
+  ButtonTypes,
+  SizeOptions,
+} from "../button/button.component";
+import MinorButton from "./minor-button.component";
+import { assertStyleMatch } from "../../__spec_helper__/test-utils";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const render = (props: ButtonProps, renderer: any = shallow) => {
+  return renderer(<MinorButton {...props} />);
+};
+
+const isMinorSizesPadding: [SizeOptions, string][] = [
+  [
+    "small",
+    "var(--spacing000) var(--spacing100) var(--spacing000) var(--spacing100)",
+  ],
+  ["medium", "var(--spacing100)"],
+  ["large", "var(--spacing100)"],
+];
+
+interface Types {
+  background?: string;
+  borderColor?: string;
+  color?: string;
+}
+
+const isMinorColors: [ButtonTypes, Types][] = [
+  [
+    "primary",
+    {
+      background: "var(--colorsActionMinor500)",
+      borderColor: "var(--colorsActionMinorTransparent)",
+      color: "var(--colorsActionMinorYang100)",
+    },
+  ],
+  [
+    "secondary",
+    {
+      background: "transparent",
+      borderColor: "var(--colorsActionMinor500)",
+      color: "var(--colorsActionMinor500)",
+    },
+  ],
+  [
+    "tertiary",
+    {
+      background: "transparent",
+      borderColor: "transparent",
+      color: "var(--colorsActionMinor500)",
+    },
+  ],
+  [
+    "darkBackground",
+    {
+      borderColor: "transparent",
+    },
+  ],
+];
+
+describe("Button Minor", () => {
+  describe("Padding", () => {
+    it.each(isMinorSizesPadding)(
+      "when size is %s the padding is %s",
+      (size, padding) => {
+        const wrapper = mount(<MinorButton size={size}>Foo</MinorButton>);
+        assertStyleMatch({ padding }, wrapper);
+      }
+    );
+  });
```

Comment:
```suggestion
    it.each(isMinorSizesPadding)(
      "when size is %s the padding is %s",
      (size, padding) => {
        const wrapper = mount(<MinorButton size={size}>Foo</MinorButton>);
        assertStyleMatch({ padding }, wrapper);
      }
    );
```
Personally I would remove the `describe` block here, since we have a small number of tests in this file and the padding tests themselves are quite self-explanatory.